Converter.kt

```diff
@@ -0,0 +1,29 @@
+fun main() {
+  //I defined my variable as fahrenheitValue here, 
+  //giving the variable a list of 5 values which I just chose multiples of 32.
+    val fahrenheitValue = listOf(32.0, 64.0, 96.0, 128.00, 160.00)
+
+  //On this line, I told the code that it needed to go through each value in the list above and convert it to celsius. 
+    for (fahrenheit in fahrenheitValue) {
+        
+  //Fahrenheit to Celsius converstion formula.
+    val celsius = (fahrenheit - 32) * 5 / 9
+
+  //After it was converted, println tells the code to print the final product so you can see the conversion. 
+        println("$fahrenheit°F is equal to %.2f°C".format(celsius))
```

Comment:
Your decimal formatting is the first one is one decimal point where as the second one is to the hundreth decimal